.github/changes-filter.yaml

```diff
@@ -9,6 +9,7 @@ docs:
   - '**/*.md'
   - '.github/changes-filter.yaml'
   - '.github/ISSUE_TEMPLATE/**'
+  - 'src/owasp-dependency-check-suppressions.xml'
```

Comment:
this is wrong, it should go on "need_owasp" section

Reply:
@nicoloboschi It's already in the "need_owasp` section, I think it's also no need to run tests, so I put it in here.